tests/unit/test_cassette_patterns.py

```
@@ -0,0 +1,434 @@
+
+
+# ---------------------------------------------------------------------------
+# Bad-cassette regression sanity check (A3 cooperation; inline payloads so
+# this test does not depend on T8.A3 filesystem fixtures landing first)
+# ---------------------------------------------------------------------------
+
+
+def test_bad_cassette_byte_count_payload_with_email_is_flagged() -> None:
+    """A synthetic bad-cassette body with a leaked email is flagged."""
+    body = '12\n{"u":"alice@gmail.com"}\n'
+    ok, leaks = is_clean(body)
```

Ruff (0.15.12)

[warning] Unpacked variable `leaks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)